app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controller");
+
+const app = express();
+
+app.get("/api/", (req, res) => {
+  res.status(200).send({ msg: "NC-NEWS Server is up and running..." });
+});
+
+app.get("/api/topics", getAllTopics);
+
+app.use((err, req, res, next) => {
```

Comment:
Nice implementation of an error handler here. I would however suggest you have jumped the gun - there is no test associated with what this code is doing. Also, as this is your only error handler in the middleware chain, `next` doesn't do anything here as there isn't another one.
Consider refactoring this to be a 500 error handler for now